src/Features/Core/Portable/Completion/Providers/Snippets/AbstractSnippetCompletionProvider.cs

```diff
@@ -88,11 +88,12 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
             foreach (var snippetData in snippets)
             {
                 var completionItem = SnippetCompletionItem.Create(
-                    displayText: snippetData.DisplayName,
+                    displayText: snippetData.SnippetIdentifier,
                     displayTextSuffix: "",
                     position: position,
                     snippetIdentifier: snippetData.SnippetIdentifier,
                     glyph: Glyph.Snippet,
+                    inlineDescription: snippetData.Description,
```

Comment:
Unrelated to this PR, but I think we should provide description for snippets (tooltips, not inline description)